marimo/_utils/health.py

```diff
@@ -2,16 +2,43 @@
 from __future__ import annotations
 
 import importlib.metadata
+import os
 import subprocess
 import sys
-from typing import Optional
+import time
+from typing import Optional, TypedDict
 
 from marimo import _loggers
 
 LOGGER = _loggers.marimo_logger()
 
 TIMEOUT = 10  # seconds
 
+# Module-level state for cgroup CPU percent calculation (like psutil does)
+_last_cgroup_cpu_sample: Optional[tuple[int, float]] = (
```

Comment:
okay to make this capitalized like other parts in the codebase
```suggestion
_LAST_CGROUP_CPU_SAMPLE: Optional[tuple[int, float]] = (
```